src/components/loadable.cjsx

```diff
@@ -49,5 +50,27 @@ module.exports = React.createClass
       render()
     else if isFailed()
       renderError()
+      
     else
       render()
+
+
+RefreshButton = React.createClass
+  displayName: 'RefreshButton'
+
+  reloadPage: ->
+    location.reload()
+
+  render: ->
+    if React?
+      <div className="refresh-button">
+        <BS.Button onClick={@reloadPage}>
```

Comment:
I think this needs to be an `<a href={window.location}>` in the case of React no longer re-rendering (due to Invariant errors or something)